Razmik_Kirakosyan/python/perfect_number/perfect_num.py

```diff
@@ -0,0 +1,12 @@
+def get_divisors(n):
+	x = 0
+	for i in range(1,n):
+		if n%i == 0:
+			x+=i
+	if n == x:
+		print ("This number is perfect")
+	else:
+		print ("This number isn't perfect")
+
+n = int(input())
+get_divisors(n)
```

Comment:
- No validation for invalid inputs(string, -1)
![image](https://user-images.githubusercontent.com/80816144/119725691-4bff1900-be81-11eb-891a-543abd4cfa6c.png)
- 0 is not perfect